src/core_modules/capture-core/utils/uid/generateUID.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+export const generateUID = (): string => {
```

Comment:
Any reason you make our own Id generator and don't use the one from the `uuid` library?

Reply:
This is to get an Id that looks like the others in the dhis2-system. It's the function that was used in TC. I guess it's only cosmetic, but not sure how the system would react when using 32 characters (uuid) vs 8 characters (DHIS2). Do you prefer to use the uuid-library?

Reply:
I was not aware that this is the shape of id used by the old tracker and across the dhis2-system. Better leave it as it is then. Thanks for the explanation! 😄